scheduler/cloudformation/cloudformation.go

```diff
@@ -512,6 +633,16 @@ func (s *Scheduler) stack(stackName *string) (*cloudformation.Stack, error) {
 	return resp.Stacks[0], nil
 }
 
+// output returns the cloudformation.Output that matches the given key.
+func (s *Scheduler) output(stack *cloudformation.Stack, key string) (output *cloudformation.Output) {
```

Comment:
This doesn't depend on `Scheduler`, so we can just move it into a `func output(...)`